src/components/Api.js

```diff
@@ -0,0 +1,84 @@
+export default class Api {
```

Comment:
Needs correction: I can't add new cards `TypeError: Failed to fetch
    at Api.addCard (Api.js:40:12)
    at PopupWithForm.handleAddCardSubmit [as _handleFormSubmit] (index.js:63:6)
    at HTMLFormElement.<anonymous> (PopupWithForm.js:32:12)`

Reply:
I check both the cards and the profile and I had no issues on my side. 
I changed the profile and the avatar, plus I added a new card as proof.

Reply:
Sorry, now everything works fine

Reply:
Hello, I know it is kind of late to reach out, but I was wondering if I made a mistake. It is saying that my project wasn't approved, yet I am now on sprint 10. I was just reaching out to see if there was a error on my side.